android/app/build.gradle

```diff
@@ -223,6 +223,13 @@ repositories {
 }
 
 dependencies {
+    compile project(':react-native-splash-screen')
```

Comment:
you can delete all these additions.

React Native link command still uses old gradle syntax, we've migrated to gradle 4. You can see below that all these dependencies are present

Reply:
Will do